### PR TITLE
fix a couple small issues

### PR DESCRIPTION
**Pull request by grondo**
This PR just addresses a couple small items:
 * do not run pr-validator except for PRs
 * add missing reference to RFC 31 in index

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -216,6 +216,12 @@ This specification describes the Flux implementation of the Hostlist Format
 
 This specification describes the Flux job urgency parameter.
 
+:doc:`31/Job Constraints Specification <spec_31>`
```
Comment (chu11) [inline index.rst:219]: commit message typo "toctree"

**Review by chu11 — APPROVED** [review]
LGTM, just saw the one typo

File: index.rst (hunk 1 above)
Comment (grondo) [reply index.rst:219]: Thanks!

Comment (grondo) [reply index.rst:219]: wait, that's what it is called in rst/sphinx: `toctree` (or did I look in the wrong spot?)

Comment (chu11) [reply index.rst:219]: oops, i thought you meant "doctree" .. nevermind :-)

**Pull request merged**